flight/Modules/VtolPathFollower/vtol_follower_control.c

```diff
@@ -201,48 +387,7 @@ int32_t vtol_follower_control_endpoint(const float dT, const float *hold_pos_ned
 int32_t vtol_follower_control_land(const float dT, const float *hold_pos_ned,
```

Comment:
Yes-- I saw the completed flag.  I still think it would make sense to combine this with endpoint following.